mongo_orchestration/servers.py

```diff
@@ -191,7 +191,12 @@ def version(self):
             command = (self.name, '--version')
             stdout, _ = subprocess.Popen(
                 command, stdout=subprocess.PIPE).communicate()
-            match = re.search(self.version_patt, str(stdout))
+            version_output = str(stdout)
+            match = re.search(self.version_patt, version_output)
+            if match is None:
+                raise ServersError(
+                    'Could not determine version of %s from string: %s'
+                    % (self.name, version_output))
             version_string = match.group('version')
```

Comment:
If there's no match, shouldn't this raise an AttributeError, because None has no method "group"? I don't see that exception in the log, so I'm not certain that raising ServersError will add info to the log either.